buildscripts/healthcheck.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+#
+# Minio Cloud Storage, (C) 2017 Minio, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+HealthCheckMain () {
+    # get the http response code
+    http_response=$(curl -s -o /dev/null -I -w "%{http_code}" http://127.0.0.1:9000/minio/index.html)
```

Comment:
Perhaps keep `ip, resource` as part of the constants.. and then reference here..